test/controllers/issueish-list-controller.test.js

```diff
@@ -85,22 +85,40 @@ describe('IssueishListController', function() {
     assert.deepEqual(view.prop('issueishes').map(issueish => issueish.getNumber()), [11, 13, 12]);
   });
 
-  it('opens reviews', async function() {
-    const atomEnv = global.buildAtomEnvironment();
-    sinon.stub(atomEnv.workspace, 'open').resolves();
-    sinon.stub(reporterProxy, 'addEvent');
-    const pr = createPullRequestResult({number: 1337});
-    Object.assign(pr.repository, {owner: {login: 'owner'}, name: 'repo'});
-    const wrapper = shallow(buildApp({
-      workspace: atomEnv.workspace,
-      endpoint: getEndpoint('github.com'),
-      results: [pr],
-    }));
+  describe('openOnGitHub', function() {
+    const url = 'https://github.com/atom/github/pull/2084';
+
+    it('calls shell.openExternal with specified url', async function() {
+      const wrapper = shallow(buildApp());
+      sinon.stub(shell, 'openExternal').callsArg(2);
+
+      await wrapper.instance().openOnGitHub(url);
+      assert.equal(shell.openExternal.args[0][0], url);
+    });
 
-    await wrapper.find('IssueishListView').prop('openReviews')();
-    assert.isTrue(atomEnv.workspace.open.called);
-    assert.isTrue(reporterProxy.addEvent.calledWith('open-reviews-tab', {package: 'github', from: 'BareIssueishListController'}));
+    it('fires `open-issueish-in-browser` event upon success', async function() {
+      const wrapper = shallow(buildApp());
+      sinon.stub(shell, 'openExternal').callsArg(2);
+      sinon.stub(reporterProxy, 'addEvent');
 
-    atomEnv.destroy();
+      await wrapper.instance().openOnGitHub(url);
+      assert.equal(reporterProxy.addEvent.callCount, 1);
```

Comment:
`strictEqual` here would make the test more resilient than just `equal`.

Reply:
Ah yes, overlooked that while copypastaing. Thanks!